ext/mysqli/mysqli_api.c

```diff
@@ -2254,7 +2254,7 @@ PHP_FUNCTION(mysqli_ssl_set)
 		}
 	}
 
-	mysql_ssl_set(mysql->mysql, ssl_parm[0], ssl_parm[1], ssl_parm[2], ssl_parm[3], ssl_parm[4]);
+	mysql_ssl_set(mysql->mysql, ssl_parm[0], ssl_parm[1], ssl_parm[2], ssl_parm[3], ssl_parm[4], NULL);
```

Comment:
The appropriate flag should be `#ifdef PDO_USE_MYSQLND`. It's unfortunate that this would be mysqlnd-only functionality, but I guess there's no way around that.